tests/iio_attr.c

```diff
@@ -126,6 +146,8 @@ static struct iio_context * autodetect_context(void)
 		printf("Using auto-detected IIO context at URI \"%s\"\n",
 				iio_context_info_get_uri(info[0]));
 		ctx = iio_create_context_from_uri(iio_context_info_get_uri(info[0]));
+		if(gen_code)
```

Comment:
it's a good idea to keep the code style in-sync with the rest of the file;
for this file there are spaces betweeb `if` and parentheses;
`if(gencode)` -> `if (gencode)`

Reply:
fixed